### PR TITLE
link to gh releases page

### PR DESCRIPTION
**Pull request by sventschui**
Link to GH Releases page from Mfrom menu and from version tag

### CODE REVIEW

**Comment by FKSI** [comment]
Oh just notice that we can use yarn the web-toolkit 😱👍 thanks @sventschui

**Comment by sventschui** [comment]
A lot faster than npm. But be aware that yarn suffers from some serious bugs. If you experience issues a good ol' npm install may help ;)

File: docs/_partials/page.hbs [hunk 1]
```diff
@@ -61,7 +61,7 @@
           </div>
 
           <div class="header-title float-xs-right">
-            <span class="label label-primary">{{package.version}}</span>
+            <a class="label label-primary" href="https://github.com/axa-group/web-toolkit/releases" >{{package.version}}</a>
```
Comment (davidknezic) [inline docs/_partials/page.hbs]: Could you put this URL to the package.json? You could add a new key like `releasesUrl`.

File: docs/_partials/page.hbs [hunk 2]
```diff
@@ -99,6 +99,12 @@
           {{#with menu.side}}
             {{> menu}}
           {{/with}}
+          <li class="mobile-menu__item">
+            <a class="mobile-menu__link"
+              href="https://github.com/axa-group/web-toolkit/releases">
```
Comment (davidknezic) [inline docs/_partials/page.hbs]: When adding the URL to `package.json`, you could use it here, too.

File: yarn.lock [hunk 3]
```diff
@@ -8347,7 +7355,12 @@ yeast@0.1.2:
...
-  version "2.2.0"
-  resolved "https://registry.yarnpkg.com/zeroclipboard/-/zeroclipboard-2.2.0.tgz#a7334d8d2e20f409ee72db4336e4849cafa2bbbc"
+zip-stream@~0.8.0:
+  version "0.8.0"
+  resolved "https://registry.yarnpkg.com/zip-stream/-/zip-stream-0.8.0.tgz#236b2fe25823cb4f48e8336f5bfa743aa5ae9dbd"
+  dependencies:
+    archiver-utils "~0.3.0"
+    compress-commons "~0.4.0"
+    lodash "~3.10.1"
+    readable-stream "~2.0.0"
```
Comment (davidknezic) [inline yarn.lock]: Why did the yarn file change so much?

**Review by davidknezic — CHANGES_REQUESTED** [review]

**Review by davidknezic — APPROVED** [review]

**Comment by davidknezic** [comment]
Check it out here:
https://design.axa.com/imrovement-link-to-gh-releases/

**Comment by davidknezic** [comment]
Thanks @sventschui for the suggestion and your pr!

**Comment by sventschui** [comment]
You're welcome

**Pull request merged**